migrations/migrations/20211029205635-create-user.js

```diff
@@ -24,6 +24,11 @@ module.exports = {
       password: {
         allowNull: false,
         type: Sequelize.STRING
+      },
+      role: {
+        allowNull: false,
+        type: Sequelize.ENUM('ADMIN', 'REGULAR'),
+        defaultValue: 'REGULAR'
```

Comment:
Not like this! Create a new migration to add the new column, don't modify existing migrations